src/Fable.React.Native.fsproj

```diff
@@ -6,10 +6,10 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="Fable.Import.ReactNative.fs" />
-    <Compile Include="Fable.Helpers.ReactNative.fs" />
-    <Compile Include="Fable.Import.XmlDom.fs" />
-    <Compile Include="extra/react-native-image-picker/Fable.Import.ReactNativeImagePicker.fs" />
+    <Compile Include="Fable.ReactNative.Types.fs" />
+    <Compile Include="Fable.ReactNative.fs" />
+    <Compile Include="Fable.XmlDom.fs" />
+    <!-- <Compile Include="extra/react-native-image-picker/Fable.Import.ReactNativeImagePicker.fs" />
```

Comment:
ah urgs. didn't see this trick. which made the PR compile. lol
now I better unlist the package ;-)

Reply:
@alfonsogarciacaro I'm now trying to apply same changes to the additional modules. I'll come back to you if I see issues

Reply:
ok I made a lot of progress (see master) but I'm not sure how to do ReactNativePopupMenu - any help appreciated